modules/StreamingAPI/src/main/java/org/gephi/streaming/api/Graph2EventListener.java

```diff
@@ -0,0 +1,253 @@
+//                    graphChangeList.add(value);
+//                    
+//                } else {
+//                    throw new RuntimeException("Unrecognized graph object type");
+//                }
+//            }
+//            
+//            Map<String, Object> graphAttributes = new HashMap<String, Object>();
+//            for (AttributeValue value: graphChangeList) {
+//                graphAttributes.put(value.getColumn().getTitle(), value.getValue());
+//            }
+//            if (!graphAttributes.isEmpty()) {
+//                org.gephi.streaming.api.event.GraphEvent streamingEvent =
+//                        eventBuilder.graphEvent(ElementType.GRAPH, EventType.CHANGE, null, graphAttributes);
+//                eventHandler.handleGraphEvent(streamingEvent);
+//            }
+//            
+//            for (Map.Entry<String, List<AttributeValue>> entry: nodeChangeTable.entrySet()) {
+//                Map<String, Object> attributes = new HashMap<String, Object>();
+//                for (AttributeValue value: entry.getValue()) {
+//                    if (value.getColumn().getIndex() != PropertiesColumn.NODE_ID.getIndex()
+//                       && value.getColumn().getIndex() != PropertiesColumn.EDGE_ID.getIndex())
+//                        attributes.put(value.getColumn().getTitle(), value.getValue());
+//                }
+//                
+//                if (!attributes.isEmpty()) {
+//                    org.gephi.streaming.api.event.GraphEvent streamingEvent =
+//                            eventBuilder.graphEvent(ElementType.NODE, EventType.CHANGE, entry.getKey(), attributes);
+//                    eventHandler.handleGraphEvent(streamingEvent);
+//                }
+//            }
+//
+//            for (Map.Entry<String, List<AttributeValue>> entry: edgeChangeTable.entrySet()) {
+//                Map<String, Object> attributes = new HashMap<String, Object>();
+//                for (AttributeValue value: entry.getValue()) {
+//                    if (value.getColumn().getIndex() != PropertiesColumn.NODE_ID.getIndex()
+//                       && value.getColumn().getIndex() != PropertiesColumn.EDGE_ID.getIndex())
+//                        attributes.put(value.getColumn().getTitle(), value.getValue());
+//                }
+//
+//                if (!attributes.isEmpty()) {
+//                    org.gephi.streaming.api.event.GraphEvent streamingEvent =
+//                            eventBuilder.graphEvent(ElementType.EDGE, EventType.CHANGE, entry.getKey(), attributes);
+//                    eventHandler.handleGraphEvent(streamingEvent);
+//                }
+//            }
+//
+//            break;
+//        }
+    }
+
+    private Map<String, Object> getNodeAttributes(Node node) {
+        Map<String, Object> attributes = new HashMap<String, Object>();
+        for (Column column: node.getAttributeColumns()) {
+            if (column.getIndex() != GraphStoreConfiguration.ELEMENT_ID_INDEX) {
```

Comment:
I would suggest to replace this by !column.isProperty(). That would ignore the `id` but also the `label` and the `timeset` if defined.